src/witness/svcompSpec.ml

```diff
@@ -23,6 +27,16 @@ let of_string s =
         UnreachCall f
       else
         failwith "Svcomp.Specification.of_string: unknown global not expression"
+  else if Str.string_match regexp s 0 then
```

Comment:
As an info:
* This part here has been changed in a separate branch, used for the SV-COMP "Memory Safety" benchmarks
* It will be added to Goblint as part of another PR